interceptors/src/main/java/uk/gov/moj/sdt/interceptors/service/RequestDaoService.java

```diff
@@ -36,7 +37,7 @@ public void persistRequest() {
         // Prepare log message for Hibernate.
         final IServiceRequest serviceRequest = new ServiceRequest();
         serviceRequest.setBulkCustomerId(extractBulkCustomerId(rawXml));
-        serviceRequest.setRequestPayload(rawXml);
+        serviceRequest.setRequestPayload(rawXml == null ? null : rawXml.getBytes(StandardCharsets.UTF_8));
```

Comment:
Have spotted that null input is handled slightly differently elsewhere.  In civil-sdt BulkSubmissionService any null input is changed to an empty string instead - see
https://github.com/hmcts/civil-sdt/blob/master/services/src/main/java/uk/gov/moj/sdt/services/BulkSubmissionService.java#L207
We should have a consistent approach of either setting it to null or not - which one would be the best?

If this line is changed then line 80 below will need to be changed as well.

Reply:
addressed it now: 
        serviceRequest.setRequestPayload(rawXml == null ? "".getBytes() : rawXml.getBytes(StandardCharsets.UTF_8));